deploy/chart/catalog_resources/rh-operators/elasticsearch.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: elasticsearch.v0.0.1
+  namespace: placeholder
+  annotations:
+    olm-examples: '[{"apiVersion": "logging.openshift.io/v1alpha1","kind": "ClusterLogging","metadata":{"name": "example","annotations":{"io.openshift.clusterlogging.alpha/allinone": ""}},"spec": {"logStore":{"type": "elasticsearch","elasticsearch":{"replicas":1,"storage":{"emptyDir": {}}}},"visualization":{"type": "kibana","kibana":{"replicas": 1}},"curation":{"type": "curator","curator": {"schedule": "30 3 * * *"}},"collection": {"logCollection":{"type": "fluentd","fluentd":{"nodeSelector":{"logging-infra-fluentd: "true"}}}}}]'
+spec:
+  displayName: Elasticsearch
+
+  description: |
+    The Elasticsearch Operator for OKD provides easy means for configuring and managing your Elasticsearch cluster.
+
+    Once installed, the Elasticsearch Operator provides the following features:
+    * **Create/Destroy**: Easily launch and create an elasticsearch cluster in the `openshift-logging` namespace.
+    * **Simple Configuration**: Configure your Elasticsearch's structure like node roles and node replica count easily.
+
+  keywords: ['elasticsearch', 'logging', 'aggregated', 'efk']
+
+  maintainers:
+  - name: Red Hat
+    email: aos-logging@redhat.com
+
+  provider:
+    name: Red Hat
+
+  links:
+  - name: Elastic
+    url: https://www.elastic.co/
+  - name: Documentation
+    url: https://github.com/openshift/elasticsearch-operator/blob/master/README.md
+  - name: Elasticsearch Operator
+    url: https://github.com/openshift/elasticsearch-operator
+
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+      - serviceAccountName: elasticsearch-operator
+        rules:
+        - apiGroups:
+          - logging.openshift.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - pods
```

Comment:
Need to add `pods/exec` here - see https://github.com/openshift/elasticsearch-operator/pull/41/files#diff-d9c51af46a881aab91ef7ababe2079dfR17

Reply:
done.